org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/AbstractMatcher.java

```diff
@@ -25,6 +26,35 @@
 
 	AbstractInsnNode cursor;
 
+	/**
+	 * Sets {@link #cursor} to first instruction of method if it is
+	 * <code>ALOAD 0</code>, otherwise sets it to <code>null</code>.
+	 */
+	void firstIsLoadThis(final MethodNode methodNode) {
```

Comment:
Strictly speaking the method name only applies to  instance methods. For static methods it would be the first argument.

Reply:
@marchof renamed into `firstIsALoad0`.